### PR TITLE
More refactorings in Perf Test tool

### PR DESCRIPTION
**Pull request by basoundr**
This is the last part of the CR changes that will be part of the #9948 PR

@KevinH-MS @TyOverby @rchande @VSadov Could you please review this change?

Also you will notice that I have made the Case-Sensitive folder name rename using 2 commits since git is unhappy about renaming a folder to a Case-Sensitive change

### CODE REVIEW

**Comment by basoundr** [comment]
@KevinH-MS @TyOverby @rchande @VSadov  ping?

File: src/Test/Perf/util/traceManager_util.csx [hunk 1]
```diff
@@ -107,11 +107,11 @@ class NoOpTraceManager : ITraceManager
 class TraceManager : ITraceManager
 {
     private readonly ScenarioGenerator _scenarioGenerator;
+    private readonly int _iterations;
 
-    private string _cpcPath = "CPC.exe";
+    private string _cpcPath;
```
Comment (KevinH-MS) [inline src/Test/Perf/util/traceManager_util.csx]: readonly?

**Comment by KevinH-MS** [comment]
:+1:

**Comment by TyOverby** [comment]
LGTM pending a squash.

**Pull request merged**